doc/oap.md

```diff
@@ -5,64 +5,21 @@ It allows issuance and transfer of user-created assets.
 
 The Open Assets Protocol is designed for Bitcoin blockchain, but it is possible to adapt to Tapyrus blockchain.
 
-To support it in electrs, some API should be updated or added.
+To support it in electrs-tapyrus, some API should be added.
 
-## Updated API
-
-### blockchain.scripthash.get_balance
-
-Return the confirmed and unconfirmed balances of a script hash.
-
-#### Signature
-
-blockchain.scripthash.get_balance(scripthash)
-
-scripthash
-
-The script hash as a hexadecimal string.
-
-#### Result
-
-A dictionary with keys confirmed, unconfirmed and assets. The value of confirmed and unconfirmed is the appropriate balance in coin units as a string.
-
-#### Result Example
-
-```
-{
-  "confirmed": "1.03873966",
-  "unconfirmed": "0.236844"
-  "colored": [
-    {
-      "asset_id": "ALn3aK1fSuG27N96UGYB1kUYUpGKRhBuBC",
-      "confirmed_asset_quantity": 100,
-      "unconfirmed_asset_quantity": 10,
-    },
-    {
-      "asset_id": "ALVYPzjZjSLoT4xAHJ7x38RcHWNatP4Mba",
-      "confirmed_asset_quantity": 100,
-      "unconfirmed_asset_quantity": 10,
-    }
-  ],
-  "uncolored": {
-    "confirmed": 1.01,
-    "unconfirmed": 0.0001,
-  }
-}
-```
-
-### blockchain.scripthash.listunspent
+## blockchain.openassets.scripthash.listunspent
```

Comment:
There are two separate APIs to get a confirmed UTXO, listuncorloredunspet and listcoloredunsent, but this API is only one?

Reply:
`blockchain.openassets.scripthash.listcorloredunspent` returns colored utxo only.
`blockchain.openassets.scripthash.listuncorloredunspent` returns uncolored utxo only.
`blockchain.openassets.scripthash.listunspent` returns both.

These APIs are about the Open Assets Protocol. 
I'm going to propose new APIs that cover native token implemented in version 0.5.0 after this pull request.

Reply:
I see.